R/AllGenerics.R

```diff
@@ -672,7 +673,7 @@ FRASER.results <- function(object, sampleIDs, fdrCutoff, zscoreCutoff,
         ans <- lapply(seq_along(sampleChunks), function(idx){
             message(date(), ": Process chunk: ", idx, " for: ", type)
             sc <- sampleChunks[[idx]]
-            tmp_x <- object[,sc]
+            tmp_x <- object[,sc,,drop=FALSE]
```

Comment:
Why do you have the extra ,? They are not positional arguments so it should work without the extra missing argument or? This looks to me as wrong coding or some potential problematic code in the future.